tardis/conftest.py

```diff
@@ -211,3 +220,139 @@ def simulation_verysimple(config_verysimple, atomic_dataset):
     sim = Simulation.from_config(config_verysimple, atom_data=atomic_data)
     sim.iterate(4000)
     return sim
+
+
+# -------------------------------------------------------------------------
+# fixtures and plugins for syrupy/regression data testing
+# -------------------------------------------------------------------------
+
+pytest_plugins = "syrupy"
```

Comment:
This is also in line 94. Should one be removed?

Reply:
Thanks!